src/components/LoaderComponent.jsx

```diff
@@ -1,27 +1,32 @@
 import React from 'react';
-import { Dimmer, Loader, Segment } from 'semantic-ui-react';
+import { Container, Dimmer, Grid } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
-const LoaderComponent = ({ loadingText, size, showDimmer, dimmerStyle }) => (
-  <Segment>
-    <Dimmer active={showDimmer} style={dimmerStyle}>
-      <Loader inverted size={size}>{loadingText}</Loader>
+import '../_css/LoaderComponent.css';
```

Comment:
Isn't this supposed to be `.scss`?

Reply:
It should be `.css`. Importing `.scss` will lead to styles not being applied since the browser cannot compile the it to css